### PR TITLE
Fix ValidDocsRule

### PR DESCRIPTION
**Pull request by diogoguimaraes**
Fixes small bug on the `ValidDocsRule`. Added a few more tests too.

### CODE REVIEW

**Comment by jpsim** [comment]
Looks like `testValidDocs` is failing with this change.

File: Source/SwiftLintFramework/Rules/ValidDocsRule.swift [hunk 1]
```diff
@@ -151,6 +151,12 @@ public struct ValidDocsRule: ConfigProviderRule {
             "\n/// - parameter param3: this is a void closure too" +
             "\nfunc a(param: () -> Void, param2: (parameter: Int) -> Void, " +
             "param3: (parameter: Int) -> Void) {}",
+            "/// docs\n/// - parameter param: this is a void closure" +
+            "\n/// - returns: Foo<Void>" +
```
Comment (jpsim) [inline Source/SwiftLintFramework/Rules/ValidDocsRule.swift]: can you please indent multiline strings after the first line? It makes it easier to tell which strings belong together.

**Comment by diogoguimaraes** [comment]
I'm on it ;)

**Comment by diogoguimaraes** [comment]
@jpsim sorry for this delay, can you please check if the indentation is ok now?
Thank you.

**Comment by jpsim** [comment]
This is good to go once CI passes. Thanks for your continued work on this, @diogoguimaraes!

I'm assuming you need another release with this? :grimacing:

**Comment by diogoguimaraes** [comment]
No problem, I'm glad to contribute!
It would be great to have a release with this :smile:

**Comment by jpsim** [comment]
Released 0.7.2 with this: https://github.com/realm/SwiftLint/releases/tag/0.7.2

Homebrew PR: https://github.com/Homebrew/homebrew/pull/48571

**Pull request merged**